docs/VoiceAttack-Integration.md

```diff
@@ -346,8 +346,10 @@ This function uses EDDI's voice to read a script. It takes one mandatory and two
 - 'Script' (text variable) is a mandatory parameter containing the script to be read. 
 - 'Priority' (integer variable) is an optional parameter defining the priority of the invoked speech (defaults to 3).
 - 'Voice' (text variable) is an optional parameter defining the name of the voice you want to use.  Note that when you set this variable it will continue to be used until you unset it, at which point EDDI will use the voice configured in its text-to-speech settings.
+- 'Volume' (integer variable) is an optional parameter defining the percentage of the configured voice volume to use.  Note that when you set this variable it will continue to be used until you unset it or reset it to 100, at which point EDDI will use the voice volume configured in its text-to-speech settings.
```

Comment:
As stated above, I'm not sure we want `Volume` to be this sticky?

Reply:
Maybe it should stick just until the end of the VoiceAttack command (or script, if invoked from Cottle)?